src/aux_utils.cpp

```diff
@@ -1103,7 +1195,19 @@ namespace diskann {
 
     diskann::build_merged_vamana_index<T>(
         data_file_to_use.c_str(), diskann::Metric::L2, L, R, p_val,
-        indexing_ram_budget, mem_index_path, medoids_path, centroids_path);
+        indexing_ram_budget, mem_index_path, medoids_path, centroids_path,
+        selection_stragegy_of_starting_points, num_starting_points);
+
+    auto old_starting_points_id_file =
```

Comment:
What strategy are you using when indices are merged?

Reply:
Select start points from each shards equally. For example, we plan to select 100 start points totally and we have 10 shards. Then we could select 10 start points from each shard, and totally we get 100 start points

Reply:
But isnt the graph in each partition built to navigate from all 100 points? Would it be better to spread the overall start point budget among the shards?

Reply:
do you mean that each shard selection 100 start points, then we will have 1000 start points totally?